webviz_ert/models/ensemble_model.py

```diff
@@ -98,7 +101,18 @@ def parameters(
         self,
     ) -> Optional[Mapping[str, ParametersModel]]:
         if not self._parameters:
-            parameter_names = self._data_loader.get_ensemble_parameters(self._id)
+            parameter_names = []
+            # TODO Remove except code path after PR https://github.com/equinor/ert-storage/pull/180 is merged
```

Comment:
I think TODO's are more likely to be picked up in issues, then in the code ;)

Reply:
Generally, that is right but the point of this TODO was to remind me that there is still something to be solved and removed before having a final version of the code and merging it.... like a TODO is meant to be right :thinking: ?

Reply:
Ah, so this is not ready for merging yet then? :)

Reply:
It is now that the TODO was dealt with